pandas/core/internals/blocks.py

```diff
@@ -373,8 +373,7 @@ def apply(self, func, **kwargs) -> list[Block]:
         apply the function to my values; return a block if we are not
         one
         """
-        with np.errstate(all="ignore"):
```

Comment:
and we don't have warnings?

Reply:
i see 4 in the tests, all coming from np.sqrt or np.log with negative numbers, which it isn't obvious to me we _should_ suppress

Reply:
that's fine but https://github.com/pandas-dev/pandas/pull/43609/checks?check_run_id=3626217065

need to catch in the test then (which is more explict).